openslides/utils/autoupdate.py

```diff
@@ -461,41 +120,48 @@ def __call__(self, request: Any) -> Any:
         response = self.get_response(request)
 
         bundle = autoupdate_bundle.pop(thread_id)  # type: Dict[str, CollectionElement]
-        # If currently there is an open database transaction, then the
-        # send_autoupdate function is only called, when the transaction is
-        # commited. If there is currently no transaction, then the function
-        # is called immediately.
-        transaction.on_commit(lambda: send_autoupdate(bundle.values()))
```

Comment:
Please let us talk about the transaction problem via phone call so I can understand it better.

Reply:
Sure. See also: https://github.com/django/channels/issues/1110